react/Spinner/Readme.md

```diff
@@ -0,0 +1,49 @@
+Show a spinner when loading something.
+
+### Default 
+
+```
+<Spinner />
+```
+
+### Color
+
+```
+<div>
+  blue: <Spinner color='blue' />
+  grey: <Spinner color='grey' />
+  white: <Spinner color='white' />
+  red: <Spinner color='red' />
+</div>
+```
```

Comment:
I cannot see colored spinner, look at the screenshot below:

![](https://i.imgur.com/VFNdYDY.png)

Reply:
It is a bug from the Spinner

Reply:
https://github.com/cozy/cozy-ui/issues/197